w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
+    starter_pokemon = random.choice(pokemons)
+    print(starter_pokemon)
+
+
+'''
+ This Is The Way
+ Write a function that gets a path to the folder,
+ and returns the list of all files that begin with the
+ "startsWith" letter sequence in that folder.
+'''
+
+
+def startsWith_files(folder, startsWith):
+    list_files = os.listdir(folder)
+    list_deep_files = []
+    for file in list_files:
+        if file.startswith(startsWith):
+            list_deep_files.append(file)
+    return list_deep_files
+
+
+# -----------------------------------------------------------------------#
```

Comment:
remove this  ------------------

Reply:
delete.
how is the correct way to separate different area?